what_is_a_gdg/promoting.md

```diff
@@ -8,5 +8,5 @@ You can create the classic chevron type logo by using this generator http://gdg-
 ## Explain what a GDG is
 > Google Developer Groups are groups of people that meet regularily and learn, discuss and exchange about the latest technologies used to create and develop new IT products.
 
-## Exchange with journalists/press
-Most successful exchange happens if you have a story to tell, content that is interesting to read.
+## Exchange with journalists/ press
```

Comment:
I guess it depends with one's grammar. However, http://www.grammarly.com/handbook/punctuation/other-punctuation-marks/4/slash/ informs me that in an informal setting 'journalists/press' should be the right format.